File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/test/java/simulacionBandaMusica/models/CantanteTest.java
package simulacionBandaMusica.models;

import org.junit.jupiter.api.Test;
import simulacionBandaMusica.factories.CantanteFactory;

import static org.junit.jupiter.api.Assertions.assertEquals;

class CantanteTest {

    @Test
    void respirarCantanteTest() {
        Cantante cantanteTest = CantanteFactory.create();
        String correctMessage = "Estoy respirando como cantante con una capacidad pulmonar de " + cantanteTest.capacidadPulomar;
        assertEquals(correctMessage, cantanteTest.respirarCantante());
    }

}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/test/kotlin/simulacionBandaMusica/models/CantanteTest.kt
package simulacionBandaMusica.models

import org.junit.jupiter.api.Test
import simulacionBandaMusica.factories.CantanteFactory
import kotlin.test.assertEquals

internal class CantanteTest {
    @Test
    fun respirarCantanteTest() {
        val cantanteTest: Musico.Cantante = CantanteFactory.create()
        val correctMessage =
            "Estoy respirando como cantante con una capacidad pulmonar de " + cantanteTest.capacidadPulomar
        assertEquals(correctMessage, cantanteTest.respirarCantante())
    }
}